src/compounds/RequestStats/RequestStats.jsx

```diff
@@ -1,56 +1,62 @@
 import React from 'react'
+import { Card } from 'react-bootstrap'
 import PropTypes from 'prop-types'
-import TextBox from '../../components/TextBox/TextBox'
-import Title from '../../components/Title/Title'
 import './request-stats.css'
 
-const RequestStats = ({ billingAddress, createdAt, deadline, detailsColor, projectCode, shippingAddress, headerColor }) => {
-  const titleStyle = { color: headerColor }
-  const textStyle = { color: detailsColor }
+const RequestStats = ({ addClass, billingInfo, createdAt, deadline, shippingInfo }) => {
+  const shippingAddress = shippingInfo.address.split(/\n/)
+  const billingAddress = billingInfo.address.split(/\n/)
 
   return (
-  <article className='request-stats'>
-    <div className='mb-2'>
-      <Title title='Request Info' size='small' />
-    </div>
-    <div className='mb-2'>
-      <Title title='Date of request' size='x-small' style={titleStyle} />
-      <TextBox text={createdAt} style={textStyle} />
-    </div>
-    <div className='mb-2'>
-      <Title title='Proposals required by' size='x-small' style={titleStyle} />
-      <TextBox text={deadline} style={textStyle} />
-    </div>
-    <div className='mb-2'>
-      <Title title='Project code' size='x-small' style={titleStyle} />
-      <TextBox text={projectCode} style={textStyle} />
-    </div>
-    <div className='mb-2'>
-      <Title title='Shipping address' size='x-small' style={titleStyle} />
-      <TextBox text={shippingAddress} style={textStyle} />
-    </div>
-    <div>
-      <Title title='Billing address' size='x-small' style={titleStyle} />
-      <TextBox text={billingAddress} style={textStyle} />
-    </div>
-  </article>
-)}
+    <Card className='request-stats w-25'>
+      <Card.Header className={`${addClass}`}>
+        <Card.Title className='mb-0'>Request Info</Card.Title>
+      </Card.Header>
+      <Card.Body>
+        <Card.Subtitle>Date of request</Card.Subtitle>
+        <Card.Text className='mb-4'>{createdAt}</Card.Text>
+        <Card.Subtitle>Proposals required by</Card.Subtitle>
+        <Card.Text className='mb-4'>{deadline}</Card.Text>
+        <Card.Subtitle>Shipping Address</Card.Subtitle>
+        {shippingAddress.map((line, index) => (
+          <Card.Text
+            key={shippingInfo.id}
+            className={`${(index === shippingAddress.length - 1) ? 'mb-4' : 'mb-0'}`}
+          >
+            {line}
+          </Card.Text>
+        ))}
```

Comment:
`Card.Text` is a `p` element so the line breaks in the address were being ignored. in order to have each part of the address be on its own line, each part needs it's own `p` tag.
we also only need bottom margin for the last line of the address.

the same logic applies to the billing address below.

Reply:
it might be easier to just use the bootstrap class 'text-wrap' for the address, i'm pretty sure it forces the line to wrap. Otherwise the css selector

white-space: pre-line 

i think would do the trick

https://getbootstrap.com/docs/5.0/utilities/text/#text-wrapping-and-overflow

Reply:
neither of those worked 😕 
<img width="1166" alt="image" src="https://user-images.githubusercontent.com/29032869/206737827-5d3d26b9-b595-4871-9bf8-6d7a4de4f212.png">